### ISSUE TITLE
Error sending emails by SMTP

### ISSUE THREAD

**Issue by mcm2007** [alfio-event/alf.io#1028]
**Describe the bug**
Hey, BIG problem: i cant send emails...
The configurations is the next:
SMTP Server: smtp.zoho.eu
SMTP Port: 587
SMTP:Protocol: smtp
SMTP Username: my email
SMTP Password: an password generated only for this app.
Email Sender: my email
SMTP Propertis: Idk what to inset here so it in blank
How many e-mail should be managed within 5 sec.: 10
Reply-to address: help@the rest of the email
The number of attempts when trying to sending an email (default: 10): 10 
Add additional CC when the system send notifications to the event organizer, can insert multiple email (comma separated): I also dont know what to put here so it is also in blank.

**To Reproduce**
1. Go to 'event'
2. Click on 'email logs'
3. See error: Status: ERROR

**Expected behavior**
That it sents the email...

**Screenshots**

**Desktop (please complete the following information):**
 Browser: Chrome lastet version
Also i ´ve done a SMTP check on an external website and it shows all ok.

**Comment by cbellone** [comment]
Hi, please have a look at https://github.com/alfio-event/alf.io/issues/1029#issuecomment-939306730

**Comment by mave-git** [comment]
Hi cbellone,
none of the solutions above inside [#1029 (comment)](https://github.com/alfio-event/alf.io/issues/1029#issuecomment-939306730) didn't work for me, so I gave up for now. But thinking about and: After a standard installation of my vservers (Ubuntu 20.04 Server) the default Java-Mail (libs) where not up to date. So I did manual installing out of standard repos libmail-java 1.6.2-1. And look: Mailing works!

Will open another issue because of doubling the base url within event-urls after changing to Java 13+ ...
